### ISSUE TITLE
compiling error with the latest svn (1792)

### ISSUE THREAD

**Issue by jbenden** [jbenden/aircrack-ng-migration#800]
**Reported by Jano on 31 Oct 2010 18:17 UTC**

```
besside-ng.c: In function do_wep_crack:
besside-ng.c:1895: error: ignoring return value of write, declared with attribute warn_unused_result
make[*** [besside-ng.o](1]:) Errore 1
make[uscita dalla directory /home/jano/aircrack-ng/src
make: *** [all](1]:) Errore 2
```

Bye Jano

Migrated-From: https://trac.aircrack-ng.org/ticket/800

**Comment by jbenden** [comment]
**Comment by Jano on 31 Oct 2010 18:17 UTC**

The problem occurs after this change:
http://trac.aircrack-ng.org/changeset/1789

Jano

**Comment by jbenden** [comment]
**Comment by misterx on 31 Oct 2010 18:17 UTC**

Fixed by aircrack-ng/aircrack-ng@168c9088adaca30fcfc17ebb64bf330dcacd3ffc.

**Comment by jbenden** [comment]
**Modified by misterx on 31 Oct 2010 18:17 UTC**
